cliquet/tests/support.py

```diff
@@ -118,9 +120,8 @@ def _create_thread(self, *args, **kwargs):
 @implementer(IAuthorizationPolicy)
 class AllowAuthorizationPolicy(object):
     def permits(self, context, principals, permission):
-        if USER_PRINCIPAL in principals:
-            return True
-        return False
+        # Cliquet default authz policy uses prefixed_userid.
+        return USER_PRINCIPAL in (principals + [context.prefixed_userid])
```

Comment:
I don't think so since this policy is used to allow ``USER_PRINCIPAL`` only. But I may be wrong.